pygame_menu/menu.py

```diff
@@ -1277,7 +1277,13 @@ def _draw_focus_widget(self, surface, widget):
         if widget is None or not widget.active or not self._mouse_motion_selection:
             return
         window_width, window_height = pygame.display.get_surface().get_size()
-        x1, y1, x2, y2 = widget.get_absolute_position(self._current._scroll)
+
+        rect = widget.get_rect()
+        if widget.selected and widget.get_selection_effect():
+            rect = widget.get_selection_effect().inflate(rect)
```

Comment:
Why is assumed that all widgets should inflate when focused? Or this intends to replace the removed `get_absolute_position(self, scroll_area)` method?

Reply:
This was done in the `get_absolute_position `. The reason is to have the borders of the opacity area in line with the selection borders.
But I'm not sure it is useful. Using the rect of the widget is probably enough.